.github/workflows/new_release_branch.yml

```diff
@@ -11,14 +11,17 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       environment_name: ${{ steps.check-environment.outputs.env_name }}
+      manual_trigger: ${{ steps.check-environment.outputs.is_manual }}
     steps:
       - name: Check environment
         id: check-environment
         run: |
           if [[ ${{ github.event_name }} == 'workflow_dispatch' ]]; then
             echo ::set-output name=env_name::'manual-release'
+            echo ::set-output name=is_manual::'true'
```

Comment:
Do we need this additional variable? Since `env_name` is already being set, we should be able to just check if `env_name == "manual-release"` below instead of manual_trigger.